extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-logs.js

```diff
@@ -221,4 +194,66 @@ export class AmpStoryDevToolsTabLogs extends AMP.BaseElement {
     }
     this.mutateElement(() => logsTabSelector.appendChild(statusIcon));
   }
+
+  /**
+   * @private
+   * @return {!Element}
+   */
+  createSuccessMessage_() {
```

Comment:
Since this is static it could be a top level html template.
then this method could be removed and a buildSuccessMessageTemplate could be used instead:
```
 const buildSuccessMessageTemplate = (element) => {
  const html = htmlFor(element);
  return html"<div class="i-amphtml-story-dev-tools-logs-success">
    <div class="i-amphtml-story-dev-tools-logs-success-image"></div>
    <h1>Great Job!<br />No issues found</h1>
  </div>";
};
```

The text needs to be in a text element semantics and a11y.
h1 could be used and targeted for styling. Be sure to set `margin: 0` to override the h1 default margin if that seems like a good route.

Going this route would also require 
`logsContainer.appendChild(this.buildLogsTitle_(this.errorList_.length));`
To be in `buildLogsContent_` and removed from `createErrorsList_` which might be okay since that line is repeated in both methods it might make sense in `buildLogsContent_`.

Reply:
Actually the image had the margin so `h1`'s margin can be used for that. Sending the commit soon!